app/views/api_keys/index.html.erb

```diff
@@ -0,0 +1,59 @@
+<% @title = t(".api_keys") %>
+
+<div class="t-body">
+  <div class="table">
+    <div class="row row__header">
+      <div class="cell">
+        <%= t(".name") %>
+      </div>
+      <div class="cell">
+        <%= t(".scope") %>
+      </div>
+      <div class="cell">
+        <%= t(".created_at") %>
+      </div>
+      <div class="cell">
+      </div>
+    </div>
+
+    <% for api_key in @api_keys do %>
+      <div class="row">
+        <div class="cell">
+          <%= api_key.name %>
+        </div>
+        <div class="cell">
+          <%= i18n_api_scopes api_key %>
+        </div>
+        <div class="cell">
+          <%= t(".time_ago", time: time_ago_in_words(api_key.created_at)) %>
+        </div>
+        <div class="cell">
+          <%= link_to t(".delete"),
+                      profile_api_key_path(id: api_key.id),
+                      method: :delete,
+                      class: "t-text t-link",
+                      data: { confirm: t(".confirm")} %>
+        </div>
+      </div>
+    <% end %>
+    <div class="row">
+      <div class="cell">
+      </div>
+      <div class="cell">
+      </div>
+      <div class="cell">
+      </div>
+      <div class="cell">
+        <strong>
+          <%= link_to t(".reset"),
+                      reset_profile_api_keys_path,
+                      method: :delete,
+                      class: "t-text t-link t-link--bold",
+                      data: { confirm: t(".confirm_all")} %>
+        </strong>
+      </div>
+    </div>
+  </div>
+
+  <p><%= link_to t(".new_key"), new_profile_api_key_path, class: "t-link t-link--has-arrow" %></p>
```

Comment:
Is this able to have the styles of the button? the text is kinda hiding in the page unless you notice the text